aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -660,6 +415,617 @@ The Hosting Bundle installer logs for the module are found at *C:\\Users\\%UserN
 
 The files can be found by searching for *aspnetcore* in the *applicationHost.config* file.
 
+::: moniker-end
+
+::: moniker range="= aspnetcore-2.2"
+
+The ASP.NET Core Module is a native IIS module that plugs into the IIS pipeline to either:
+
+* Host an ASP.NET Core app inside of the IIS worker process (`w3wp.exe`), called the [in-process hosting model](#in-process-hosting-model).
+* Forward web requests to a backend ASP.NET Core app running the [Kestrel server](xref:fundamentals/servers/kestrel), called the [out-of-process hosting model](#out-of-process-hosting-model).
+
+Supported Windows versions:
+
+* Windows 7 or later
+* Windows Server 2008 R2 or later
+
+When hosting in-process, the module uses an in-process server implementation for IIS, called IIS HTTP Server (`IISHttpServer`).
+
+When hosting out-of-process, the module only works with Kestrel. The module is incompatible with [HTTP.sys](xref:fundamentals/servers/httpsys).
```

Comment:
Maybe this should explain what it means to be incompatible with HTTP.sys, because I'm not really sure what we're trying to indicate when we say here.